index.html

```diff
@@ -1255,6 +1246,262 @@ <h3>
           conventions or limitations of the host operating system.
         </p>
       </section>
+      <section>
+        <h3>
+          `*_localized` members
+        </h3>
+        <p>
+          A <dfn>localizable member</dfn> is a [=manifest=] member that can be
+          localized. Each [=localizable member=] of the [=manifest=] has a
+          corresponding <code><dfn data-export="" data-dfn-for=
+          "manifest">*_localized</dfn></code> member, where `*` represents the
+          member name.
+        </p>
+        <aside class="example">
+          <p>
+            For example, the [=manifest/name=] member is a [=localizable
+            member=], and `name_localized` is its corresponding
+            [=manifest/*_localized=] member.
+          </p>
+        </aside>
+        <p>
+          A <dfn>language map</dfn> is an [=ordered map=] whose key is a
+          [=language tag=] and whose value is a [=localized value=]. The
+          <dfn>localized value</dfn> is content localized in the language given
+          by the key.
+        </p>
+        <aside class="example" title="Localizing the application name">
+          <pre class="json">
+            {
```

Comment:
In order to make it obvious why one might want to have a different 'lang' for an entry in here, perhaps we should use a company that so it would have opinions about name pronunciations in other languages.

L'Occitane is a good example - using wikipedia you can see that sometimes is uses the french version, while other times it has a translated name. For cases like English & German, screen readers should read the name in a french pronunciation. 

```suggestion
            {
              "lang": "fr",
              "dir": "ltr",
              "name":  "L'Occitane",
              "name_localized": {
                "en":  { value: "L'Occitane", "lang": "fr" },
                "de":  { value:  "L'Occitane", "lang": "fr" },
                "zh":  "歐舒丹"
                "en-GB": {"value": "L'Occitane en Provence", "lang": "fr" },
                "fr":    "L'Occitane",
                "ar":    {value: "لوکسیتان",  "dir": "rtl"}
              }
            }
```

Reply:
If it's okay to use real-world brands/examples, that's fine with me. I also have the "Just Eat" example here, where the brand even differs between de-DE and de-CH: https://github.com/w3c/manifest/pull/1101#discussion_r1676783850

@marcoscaceres WDYT?

Reply:
Sounds good, but don't use a real company name... they might not approve.

Reply:
The example below seems fine to me... @dmurph?